python-plugin/pom.xml

```diff
@@ -16,20 +16,35 @@
     <url>https://github.com/green-code-initiative/ecoCode/tree/main/python-plugin</url>
 
     <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>ecocode-rules-specifications</artifactId>
+            <version>${project.version}</version>
+            <classifier>python</classifier>
+        </dependency>
 
         <dependency>
             <groupId>org.sonarsource.python</groupId>
             <artifactId>sonar-python-plugin</artifactId>
             <type>sonar-plugin</type>
+            <scope>provided</scope>
         </dependency>
 
         <dependency>
             <groupId>org.sonarsource.sonarqube</groupId>
             <artifactId>sonar-plugin-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.sonarsource.analyzer-commons</groupId>
+            <artifactId>sonar-analyzer-commons</artifactId>
         </dependency>
+
         <dependency>
             <groupId>org.sonarsource.sonarqube</groupId>
             <artifactId>sonar-plugin-api-impl</artifactId>
+            <scope>test</scope>
```

Comment:
why test ?

Reply:
Only needed in test:
`php-plugin/src/test/java/fr/greencodeinitiative/php/PhpPluginTest.java`